### DISCUSSION TITLE
使用Xray(M+B+G+A)+Nginx配置reality不通，其他服务正常

### DISCUSSION THREAD

**Discussion by linyuancode** [lxhao61/integrated-examples#211] [Q&A] [answered]
使用Xray(M+B+G+A)+Nginx配置1，网页访问与VMESS代理均正常，且无报错，但是reality无法连接。有什么容易忽略的步骤会导致这个问题吗？

**Comment by lxhao61** [comment, marked as answer]
1、你确定仔细看了介绍、注意及备注？
2、你确定完全参考示例来实际配置的，且 reality 对应域名及证书配置正确了？
3、使用 nginx -V 命令检查是否满足所有注意及备注事项。

**Reply by linyuancode** [reply]
证书是zerossl的ec-256长度：
acme.sh --issue -d v4.lyfree.top --dns dns_cf --keylength ec-256 --force
acme.sh --issue -d osvwt.lyfree.top --dns dns_cf --keylength ec-256 --force
